glambie/const/regions.py

```diff
@@ -0,0 +1,38 @@
+
+class RGIRegion():
```

Comment:
This is a [dataclass](https://docs.python.org/3.9/library/dataclasses.html), and can be more easily defined using the dataclass decorator.

Reply:
done